JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,18 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  let hours = Math.floor(Math.floor(seconds / 60) / 60);
+  let minutes = Math.floor(seconds / 60) % 60;
+  let secs= seconds % 60;
+
+  const formatTime = (hh, mm, ss) => {
```

Comment:
I think you can define this outside too right?, not a big fan of nested functions